RULES.md

```diff
@@ -0,0 +1,21 @@
+# JavaScript-specific Rules
```

Comment:
same review feedback, do you discuss about rules managment with @jycr and the rules-specification module in ecoCode repository ?